pom.xml

```diff
@@ -261,6 +261,7 @@
     <version.org.wildfly>14.0.1.Final</version.org.wildfly>
     <version.org.wildfly.core>6.0.2.Final</version.org.wildfly.core>
 
+    <version.org.mvel>2.4.3.Final</version.org.mvel>
```

Comment:
we have mvel version property in IP BOM https://github.com/jboss-integration/jboss-integration-platform-bom/blob/8.3.2.Final/pom.xml#L326.
@mariofusco @baldimir use that dependency management. Adding such properties will confuse people.

tagging here @mbiarnes